python/samples/04.ai.a.teamsChefBot/README.md

```diff
@@ -0,0 +1,16 @@
+# Microsoft Teams Conversational Bot with AI: Teams Chef
+
+### Try the sample using Teams Toolkit for Visual Studio Code
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
```

Comment:
Please update the root sample README https://github.com/microsoft/teams-ai/blob/PYTHON/python/README.md with a link to this sample.

Reply:
updated